exercises/D-logical-operators/exercise.js

```diff
@@ -13,13 +13,31 @@ let cssLevel = 4;
 // (hint: use the comparison operator from before)
 let htmlLevelAbove5;
 let cssLevelAbove5;
-
+if (htmlLevel > 5) {
```

Comment:
Could you simplify this if statement? It's possible here to assign a boolean expression (like `htmlLevel > 5`) directly to the variable.